aldryn_faq/locale/de/LC_MESSAGES/django.po

```diff
@@ -1,292 +1,301 @@
 #: boilerplates/legacy/templates/aldryn_faq/includes/question_list.html:27
 #: boilerplates/legacy/templates/aldryn_faq/plugins/category_list.html:8
+#: templates/aldryn_faq/includes/question_list.html:28
+#: templates/aldryn_faq/includes/question_list.html:35
+#: templates/aldryn_faq/includes/ungrouped_question_list.html:19
+#: templates/aldryn_faq/plugins/category_list.html:17
 msgid "No entry found."
-msgstr ""
+msgstr "Kein Eintrage gefunden."
 
 #: boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html:51
-#: boilerplates/bootstrap3/templates/aldryn_faq/question_detail.html:41
+#: boilerplates/bootstrap3/templates/aldryn_faq/question_detail.html:35
+#: templates/aldryn_faq/includes/question_list.html:39
+#: templates/aldryn_faq/question_detail.html:28
 msgid "Back to Category"
-msgstr ""
+msgstr "Zurück zur Kategorie"
 
-#: boilerplates/bootstrap3/templates/aldryn_faq/question_list.html:13
+#: boilerplates/bootstrap3/templates/aldryn_faq/question_list.html:10
+#: templates/aldryn_faq/question_list.html:11
 msgid "Back to Overview"
-msgstr ""
+msgstr "Zurück zur Übersicht"
 
 #: boilerplates/legacy/templates/aldryn_faq/includes/question_list.html:18
 msgid "No answer available."
-msgstr ""
+msgstr "Keine Antwort verfügbar."
 
 #: boilerplates/legacy/templates/aldryn_faq/includes/question_list.html:20
 msgid "Read more"
-msgstr ""
+msgstr "Weiterlesen"
 
 #: boilerplates/legacy/templates/aldryn_faq/includes/question_list.html:32
 #: boilerplates/legacy/templates/aldryn_faq/question_detail.html:10
 msgid "Back"
-msgstr ""
+msgstr "Zurück"
 
 #: cms_app.py:14 menu.py:20
 msgid "FAQ"
-msgstr ""
+msgstr "FAQ"
 
 #: cms_appconfig.py:16
 msgid "1/2/"
-msgstr ""
+msgstr "1/2/"
 
 #: cms_appconfig.py:17
 msgid "1/question-slug/"
-msgstr ""
+msgstr "1/frage-url/"
 
 #: cms_appconfig.py:18
 msgid "category-slug/2/"
-msgstr ""
+msgstr "kategorie-url/2/"
 
 #: cms_appconfig.py:19
 msgid "category-slug/question-slug/"
-msgstr ""
+msgstr "kategorie-url/frage-url/"
 
 #: cms_appconfig.py:20
 msgid "1-category-slug/2/"
-msgstr ""
+msgstr "1-kategorie-url/2/"
 
 #: cms_appconfig.py:21
 msgid "1-category-slug/question-slug/"
-msgstr ""
+msgstr "1-kategorie-url/frage-url/"
 
 #: cms_appconfig.py:25
 msgid "Allow"
-msgstr ""
+msgstr "Erlauben"
 
 #: cms_appconfig.py:26
 msgid "Redirect to permalink (default)"
-msgstr ""
+msgstr "Weiterleitung zum Permalink (Standard)"
 
 #: cms_appconfig.py:27
 msgid "Permanent redirect to permalink"
-msgstr ""
+msgstr "Permanente Weiterleitung zum Permalink"
 
 #: cms_appconfig.py:28
 msgid "Return 404: Not Found"
-msgstr ""
+msgstr "Zeige 404: Nicht gefunden"
 
 #: cms_appconfig.py:36
 msgid "application title"
-msgstr ""
+msgstr "Applikations Titel"
 
 #: cms_appconfig.py:39
 msgid "permalink type"
-msgstr ""
+msgstr "Permalink Typ"
 
 #: cms_appconfig.py:41
 msgid ""
 "Choose the style of urls to use from the examples. (Note, all types are "
 "relative to apphook)"
```

Comment:
"Select the URL style." (I am not sure what 'all types are relative to apphook' means.)